GDJS/Runtime/variable.js

```diff
@@ -4,21 +4,36 @@
  * This project is released under the MIT License.
  */
 
+/**
+ * @typedef variableData Data representation of a GDevelop variable
+ * @property {string} [name] The name of the variable. Used if a child variable.
+ * @property {string | number} [value] The value of the variable, either string or number. Leave blank for structures.
+ * @property {variableData} [children] The children of the structure. Leave blank if value is defined.
```

Comment:
The type is not variableData, it's an array of variableData: `VariableData[]`

Reply:
🤦‍♂ I knew this. I thought I had fixed it tho

Reply:
fixed